src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface State {
+  input: string | null,
+}
+
+export class App extends React.Component<{}, State> {
+  state = {
+    input: null,
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ input: event.key });
+    });
```

Comment:
You run the same code in 2 places, create a function for this
```suggestion
    document.addEventListener('keyup', this.handleClick);
```